react-native/react/native/pinentry.wrapper.desktop.js

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from '../base-react'
+import ReactDOM from 'react-dom'
+import Pinentry from '../pinentry'
+import ipc from 'ipc'
+
+class PinentryWrapper extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {}
+    ipc.on('pinentryGotProps', props => {
```

Comment:
Agree, not Redux-y.  We do at least have a "smart" component that passes off props to the dumb component, but the smart component gets props over IPC and puts them into its local state, and doesn't use redux store/dispatch/actions/etc.

@MarcoPolo Any ideas on a better way?  What should we be trying to use from Redux here?  Thanks!